src/components/main/MoreInfos.tsx

```diff
@@ -0,0 +1,39 @@
+import { MoreInfosProps } from "../../type/types";
+import CompanyIcon from "../icons/CompanyIcon";
+import LocationIcon from "../icons/LocationIcon";
+import TwitterIcon from "../icons/TwitterIcon";
+import WebsiteIcon from "../icons/WebsiteIcon";
+import "./MoreInfos.css";
+
+function MoreInfos(props:Readonly<MoreInfosProps>) {
+  return (
+    <ul className="further-user-infos">
+      <li className={props.location ? "user-location" : "user-location not-available"}>
```

Comment:
Just add the not-available class when needed
```suggestion
      <li className={`user-location ${!props.location && 'not-available'}`}>
```

Reply:
I changed it